### PR TITLE
Helpdesk Changes April 2024 MARK-2

### PR DESCRIPTION
**Pull request by Hazard-The-AD**
- Completely removed any mentions of the Live Chat support platform.
- Minor grammar/spelling corrections.
- Deleted the `Create a Private Game` and `Cancelling Titan Rank` articles.
- Created a new article `Titan Perks FAQ` for assistance regarding all Titan perks.
- Updated Partner Programme requirements.
- Clarified 'no concrete' requirements for the Partner programme.
- Added fun emojis to make the Helpdesk less 'cold'.
- Clarified EU and Brazillian withdrawal regulations providing the specific day period.
- Added info note for anyone trying to pay with UPI/RuPay.
- Added new images to the Assets folder for the Titan Perks FAQ article.
- Updated Rules to remove mention of guild Discord channel and emphasize permanent bans depending on the severity of discrimination. Also added a hyperlink to Discord Terms of Service.

JIRA: MARK-2

### CODE REVIEW

**Comment by Hazard-The-AD** [comment]
@TobiasGrether will be checking once home.

**Review by dries-c — APPROVED** [review]

File: docs/titan-perks-faq.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## When will my payment be renewed? 💳
+Your Titan rank payment will renew 30 days from when you purchased the rank subscription. If you have purchased Titan for 1 year, 6 months, or 3 months, then your Titan rank will renew after that period. Payments are automatically renewed unless cancelled. If you wish to cancel your Titan rank, please see below.
+
+## How can I cancel my Titan rank? 🚫
+::: tip Store Login
+Remember to log in with the Xbox LIVE account you use to play NetherGames with your Titan subscription, otherwise, you may not see the option to cancel your Titan subscription.
+:::
+To cancel your Titan rank, you can visit the [NetherGames Store](https://ngmc.co/store) and click on the gear icon to open your store User Settings. This will open up the "Subscriptions" tab where you can cancel or make changes to your active Titan rank subscription.
+![image](https://github.com/HazardTheDutchAD/support/assets/20299632/0fec9a4d-7408-4341-be2b-db8740221564)
```
Comment (dries-c) [inline docs/titan-perks-faq.md:57]: This link has to be changed though

File: docs/partner-program.md [hunk 2]
```diff
@@ -1,26 +1,35 @@
-# Partner Program
+# Partner Program ⭐
```
Comment (dries-c) [inline docs/partner-program.md:1]: I'm not a big fan of unicode titles

File: docs/titan-perks-faq.md (hunk 1 above)
Comment (Hazard-The-AD) [reply docs/titan-perks-faq.md:57]: Which link specifically?

Comment (dries-c) [reply docs/titan-perks-faq.md:57]: https://github.com/HazardTheDutchAD/support/ -> https://github.com/NetherGamesMC/support/

**Pull request merged**